components/brave_shields/core/browser/brave_shields_utils.h

```diff
@@ -89,7 +89,8 @@ void SetCookieControlType(HostContentSettingsMap* map,
 ControlType GetCookieControlType(
     HostContentSettingsMap* map,
     content_settings::CookieSettings* cookie_settings,
-    const GURL& url);
+    const GURL& url,
+    PrefService* pref_service);
```

Comment:
In some places we use prefs and in others pref_service; a quick search suggests prefs is more common in Chromium, but I could be wrong :-)

Reply:
`PrefService* pref_service` naming is consistent with naming in brave_shields_utils.h.

There is also another naming `PrefService* profile_state` which is used for functions where `PrefService* local_state` parameter is presented.